### PR TITLE
Update reference instructions to account for modern taste

### PR DESCRIPTION
**Pull request by annevk**
And align the rest of the document as well.

***
<a href="https://whatpr.org/infra/494.html" title="Last updated on Oct 26, 2022, 12:10 PM UTC (413c731)">Preview</a> | <a href="https://whatpr.org/infra/494/655048a...413c731.html" title="Last updated on Oct 26, 2022, 12:10 PM UTC (413c731)">Diff</a>

### CODE REVIEW

File: infra.bs [hunk 1]
```diff
@@ -55,14 +55,12 @@ urlPrefix: https://tc39.github.io/ecma262/#; spec: ECMA-262;
 
 <h2 id=usage>Usage</h2>
 
-<p>To make use of the Infra Standard in a document titled <var>X</var>, use
-<samp><var>X</var> depends on the Infra Standard</samp>. Additionally, cross-referencing terminology
-is encouraged to avoid ambiguity.
+<p>To make use of this standard in a document titled <var>X</var>, use:
 
-<p>Specification authors are also encouraged to add their specification to the
-<a href=https://github.com/whatwg/infra/wiki/Dependent-specs>list of dependent specifications</a> in
```
Comment (annevk) [inline infra.bs:63]: I suggest we remove this at the same time. https://dontcallmedom.github.io/webdex/ is the tool to use now.

File: infra.bs [hunk 2]
```diff
@@ -55,14 +55,12 @@ urlPrefix: https://tc39.github.io/ecma262/#; spec: ECMA-262;
...
-order to help the editors ensure that any future breaking changes to the Infra Standard are
-correctly reflected by any such dependencies.
+<p><samp><var>X</var> depends on <cite>Infra</cite>. [[!Infra]]</samp>
+<!--                                                 Yo Dawg -->
```
Comment (domenic) [inline infra.bs:61]: I don't get it :(

File: infra.bs [hunk 3]
```diff
@@ -74,13 +72,14 @@ marked non-normative. Everything else is normative.
 
 <p>The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT",
 "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" are to be interpreted as described in
-RFC 2119. [[!RFC2119]]
+<cite>RFC 2119</cite>. [[!RFC2119]]
```
Comment (domenic) [inline infra.bs:75]: I tend to prefer citing the actual titles, in this case "Key words for use in RFCs to Indicate Requirement Levels". I'm not sure RFC 2119 is actually the title of a work, so if you leave it in that form, I think omitting the `<cite>` is best.

File: infra.bs [hunk 4]
```diff
@@ -820,11 +819,11 @@ and monospace font.
...
 <p class=example id=example-string-notation>"<code>Hello, world!</code>" is a string.
 
-<p class=note>This is different from how the Unicode Standard defines "code unit". In particular it
-refers exclusively to how the Unicode Standard defines it for Unicode 16-bit strings. [[UNICODE]]
+<p class=note>This is different from how <cite>Unicode</cite> defines "code unit". In particular it
+refers exclusively to how <cite>Unicode</cite> defines it for Unicode 16-bit strings. [[UNICODE]]
 
 <p>A <a>string</a> can also be interpreted as containing <a>code points</a>, per the conversion
-defined in <a>The String Type</a> section of the JavaScript specification. [[!ECMA-262]]
+defined in <a>The String Type</a> section of <cite>JavaScript</cite>. [[!ECMA-262]]
```
Comment (domenic) [inline infra.bs:826]: Again this is not the actual title, so I prefer leaving it as "the JavaScript specification"

File: infra.bs [hunk 5]
```diff
@@ -983,11 +982,11 @@ literal, we can use plainer language: <var>domain</var> <a for=string>ends with<
  <li><p>Return false.
 </ol>
 
-<p class="note">This matches the ordering used by JavaScript's <code>&lt;</code> operator, and its
-{{Array/sort()}} method on an array of strings. This ordering compares the 16-bit code units in each
-string, producing a highly efficient, consistent, and deterministic sort order. The resulting
-ordering will not match any particular alphabet or lexicographic order, particularly for
-<a>code points</a> represented by a surrogate pair. [[!ECMA-262]]
+<p class="note">This matches the ordering used by <cite>JavaScript</cite>'s <code>&lt;</code>
```
Comment (domenic) [inline infra.bs:985]: This was meant to be a reference to the language, not to a specific document.

File: infra.bs (hunk 2 above)
Comment (annevk) [reply infra.bs:61]: Infra referencing Infra seemed close enough to https://knowyourmeme.com/memes/xzibit-yo-dawg.

**Review by domenic — APPROVED** [review]

**Pull request merged**